web/views/note_view.ex

```diff
@@ -1,4 +1,8 @@
 defmodule Notox.NoteView do
   use Notox.Web, :view
 
+  def render_markdown(markdown) do
+    import Earmark
+    Earmark.to_html(markdown) |> raw
```

Comment:
Can't I strip out all the HTML from the user input? They wouldn't be able to enter HTML, but all the Markdown would work fin.